### DISCUSSION TITLE
CoveragePlot plotting additional ranges as multiple separate tracks

### DISCUSSION THREAD

**Discussion by jenellewallace** [stuart-lab/signac#1771] [Q&A] [answered]
Is there a way to plot multiple additional tracks (like from bed files or list of granges objects) on a single coveragePlot (like how it's possible to plot multiple bigwig files with a named list)? When I tried with a named list, I got this error:
`Error: unable to find an inherited method for function ‘subsetByOverlaps’ for signature ‘x = "list", ranges = "GRanges"’`

I guess I can make separate plots and join with patchwork but then my ATAC-seq data is also plotted again so I would have to combine the plots manually to make a finished figure. Would appreciate any suggestions for how to do this!

**Comment by timoast** [comment, marked as answer]
You can use the `PeakPlot` function to create additional granges tracks and combine this with the other tracks using the `CombineTracks` function

https://stuartlab.org/signac/articles/visualization#plotting-peak-coordinates
